src/shared/telemetry/defaultTelemetryClient.ts

```diff
@@ -55,6 +56,27 @@ export class DefaultTelemetryClient implements TelemetryClient {
         }
     }
 
+    public async postFeedback(feedback: TelemetryFeedback): Promise<void> {
+        try {
+            await this.client
+                .postFeedback({
+                    AWSProduct: DefaultTelemetryClient.PRODUCT_NAME,
+                    AWSProductVersion: pluginVersion,
+                    OS: os.platform(),
+                    OSVersion: os.release(),
+                    ParentProduct: vscode.env.appName,
+                    ParentProductVersion: vscode.version,
+                    Comment: feedback.comment,
+                    Sentiment: feedback.sentiment
+                })
+                .promise()
+            console.info('Successfully posted feedback')
```

Comment:
replace `console` usage with logger (applies to line 75 also)

Reply:
Thanks. Fixed the other usage in this class as well.